scripts/exgfs_wave_prep.sh

```diff
@@ -111,72 +109,70 @@
     time_rst2_end=
     DT_2_RST_WAV=
   fi
-  set +x
-  echo ' '
-  echo 'Times in wave model format :'
-  echo '----------------------------'
-  echo "   date / cycle  : ${date} ${cycle}"
-  echo "   starting time : ${time_beg}"
-  echo "   ending time   : ${time_end}"
-  echo ' '
-  set_trace
+  cat << EOF
+
+Times in wave model format :
+----------------------------
+   date / cycle  : ${date} ${cycle}
+   starting time : ${time_beg}
+   ending time   : ${time_end}
+
+EOF
 
   # Script will run only if pre-defined NTASKS
   #     The actual work is distributed over these tasks.
   if [[ -z ${NTASKS} ]]
   then
-    echo "FATAL ERROR: Requires NTASKS to be set "
-    err=1; export err;${errchk}
+    export err=1
+    err_chk "FATAL ERROR: Requires NTASKS to be set"
   fi
 
   # --------------------------------------------------------------------------- #
   # 1.  Get files that are used by most child scripts
 
-  set +x
-  echo 'Preparing input files :'
-  echo '-----------------------'
-  echo ' '
-  set_trace
+  printf "Preparing input files :\n-----------------------\n"
 
   # 1.a Model definition files
 
   rm -f cmdfile
   touch cmdfile
 
   grdINP=''
-  if [[ "${WW3ATMINP}" = 'YES' ]]; then grdINP="${grdINP} ${WAVEWND_FID}" ; fi
-  if [[ "${WW3ICEINP}" = 'YES' ]]; then grdINP="${grdINP} ${WAVEICE_FID}" ; fi
-  if [[ "${WW3CURINP}" = 'YES' ]]; then grdINP="${grdINP} ${WAVECUR_FID}" ; fi
+  if [[ "${WW3ATMINP}" == 'YES' ]]; then
+     grdINP="${grdINP} ${WAVEWND_FID}"
+  fi
+  if [[ "${WW3ICEINP}" == 'YES' ]]; then
+     grdINP="${grdINP} ${WAVEICE_FID}"
+  fi
+  if [[ "${WW3CURINP}" == 'YES' ]]; then
+     grdINP="${grdINP} ${WAVECUR_FID}"
+  fi
 
   ifile=1
 
   for grdID in ${grdINP} ${waveGRD}
   do
     if [[ -f "${COMIN_WAVE_PREP}/${RUN}.wave.mod_def.${grdID}" ]]
     then
-      set +x
       echo " Mod def file for ${grdID} found in ${COMIN_WAVE_PREP}. copying ...."
-      set_trace
       cp ${COMIN_WAVE_PREP}/${RUN}.wave.mod_def.${grdID} mod_def.${grdID}
```

Comment:
Sorry, caught another `cp` to update:
```suggestion
      cpreq ${COMIN_WAVE_PREP}/${RUN}.wave.mod_def.${grdID} mod_def.${grdID}
```

Reply:
Fixed.